web-console/src/utils/ingestion-spec.tsx

```diff
@@ -1226,6 +1226,75 @@ export function getIoConfigFormFields(ingestionComboType: IngestionComboType): F
             </>
           ),
         },
+        {
+          name: 'inputSource.properties.accessKeyId.type',
+          label: 'Access key ID type',
+          type: 'string',
+          suggestions: [undefined, 'environment', 'default'],
+          info: (
+            <>
+              <p>S3 access key for this S3 bucket.</p>
+              <p>Setting this will override the default configuration provided in the config.</p>
+            </>
+          ),
+          adjustment: (ioConfig: IoConfig) => {
+            return deepSet(
+              ioConfig,
+              'inputSource.properties.secretAccessKey.type',
+              deepGet(ioConfig, 'inputSource.properties.accessKeyId.type'),
+            );
+          },
+        },
+        {
+          name: 'inputSource.properties.accessKeyId.variable',
```

Comment:
The name is different depending on the type of password provider. It's called `variable` for `EnvironmentVariablePasswordProvider` while it's called `password` for `DefaultPasswordProvider`.

Reply:
oh I see, yup need to fix that. What about `inputSource.properties.secretAccessKey.x` ? same?

Reply:
yes. same for inputSource.properties.secretAccessKey.x